test/grinder_sdk_test.dart

```diff
@@ -128,4 +128,46 @@ main() {
       expect(newText, isNot(equals(originalText)));
     });
   });
+
+  group('grinder.sdk Analyzer', () {
```

Comment:
I think many of these tests should instead be tests of `coerceToPathList` [here](https://github.com/google/grinder.dart/blob/master/test/src/utils_test.dart).